index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start toolbar/header -->
+    <header class="toolbar">
+      <div class="logo_placeholder">
+        <a href="index.html" class="logo">Welcome</a>
+      </div>
+      <nav class="menu">
+        <button type="button" class="btn">
+          <i class="material-icons">menu</i>
+        </button>
+      </nav>
+    </header>
+    <!-- end toolbar/header -->
+
+    <!-- start main -->
+    <section class="headline background">
+      <div>
+        <h1>Hey there. I’m Radmila</h1>
+        <h2>I’m a software developer</h2>
+      </div>
+      <p>
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <a href="#"
+          ><img class="icon" src="icons/twitter.svg" alt="twitter logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/linkedin.svg" alt="linkedin logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/medium.svg" alt="medium logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/github.svg" alt="github logo"
+        /></a>
+        <a href="#"
+          ><img class="icon" src="icons/angellist.svg" alt="angellist logo"
+        /></a>
+      </div>
+      <button type="button" class="btn expand">
+        <i class="material-icons">expand_more</i>
+      </button>
+    </section>
+    <!-- end main -->
+
+    <section class="recent_work section">
+      <h1>My Recent Works</h1>
+      <div class="indicator">&nbsp;</div>
+
+      <div class="work_cards">
```

Comment:
- [ ] **Please use `ul` tag to group all the projects and `li` tag for each project.**

What this means is that you should use the `ul` tag to group all the projects together in one element. Then, inside of that `ul` element, you should use the `li` tag to display each project and its contents.

Why is it important?

Whenever you are trying to group and list things, you should consider using `ul` and `li` tags. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.